events/git-and-github.md

```diff
@@ -9,18 +9,30 @@ The workshop should cover and explain the following
 - Be able to push and pull code from Github
 - Be able to send a PR to an Open Source Repository
 
+## Timeline  
+The Workshop should be held around the time when GymK societies have their Intro-Seminar, so that we also show our presence during the same time but with a Workshop.
```

Comment:
```suggestion
The Workshop should be held around the time when Gymkhana societies have their introductory seminar, so that we also show our presence during the same time but with a workshop.
```

Same for other cases below.